src/style.css

```diff
@@ -1 +1,94 @@
 /* Styles go here */
+:root {
+  --header-height: 60px;
+  --line-height: 14.6px;
+}
+
+.header {
+  width: 1024px;
+  margin: auto 0;
+  height: var(--header-height);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  position: relative;
+  justify-content: space-around;
+}
+
+.nav {
+  position: absolute;
```

Comment:
No need to use position absolute for navigation,  you can simply add display: flex, justify-content: space-between and padding: 0 50px to header